bobtask/map.go

```diff
@@ -194,3 +195,30 @@ func (tm Map) IgnoreChildTargets() (err error) {
 
 	return nil
 }
+
+// VerifyDuplicateTargets checks if build tasks contain duplicate target names
+func (tm Map) VerifyDuplicateTargets() error {
+	targetToTasks := make(map[string][]string)
+
+	for taskName, v := range tm {
+		target, _ := v.Target()
+		if target == nil {
+			continue
+		}
+		for _, t := range target.DockerImages() {
+			targetToTasks[t] = append(targetToTasks[t], taskName)
+		}
+		for _, t := range target.FilesystemEntriesRaw() {
```

Comment:
When imports are also considered you should use `FilesystemEntries()` this let's you compare paths  in relation to the umbrella Bobfile.

Reply:
If tasks from bobfiles cannot have targets from a above directories, do we really need to do this?

Reply:
But tasks from umbrella Bobfiles can have targets in downstream directories.